modules/preparing-microshift-executable.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * microshift-install-rpm.adoc
+
+:_content-type: PROCEDURE
+[id="preparing-microshift-executable"]
+= Preparing the MicroShift executable runtime
+:context: creating-microshift-development-environment
+
+Use the following procedure to prepare the MicroShift executable runtime. 
+
+[NOTE]
+====
+When working with MicroShift based on a pre-release minor version `Y` of {OCP}, the corresponding RPM repository `rhocp-4.$Y-for-rhel-8-$ARCH-rpms` will not be available yet. As a workaround, use the `Y-1` version. 
+====
+
+.Prerequisites 
+
+* You have logged into your development virtual machine using your MicroShift user credentials. 
+* You have configured your MicroShift virtual machine. 
+* You have built MicroShift in your development virtual machine. 
+
+.Procedure
+
+. Enable the necessary repositories by entering the following command:
++
+[source,terminal]
+----
+$ sudo subscription-manager repos \
+    --enable rhocp-4.11-for-rhel-8-$(uname -i)-rpms \
+    --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+----
+
+. Install the MicroShift RPM packages by entering the following command: 
++
+[source,terminal]
+----
+$ sudo dnf localinstall -y ~/microshift/_output/rpmbuild/RPMS/*/*.rpm
```

Comment:
```suggestion
$ sudo dnf install microshift
```

The command in the dev docs installs the RPMs built from source by some of the other steps. Users should install the RPMs we ship normally.